src/treeView/TabnineTreeProvider.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable class-methods-use-this */
+import { Event, ProviderResult, TreeDataProvider, TreeItem } from "vscode";
+import { getHubStructure } from "../binary/requests/hubStructure";
+import { TABNINE_TREE_NAVIGATION_COMMAND } from "../globals/consts";
+import TabnineTreeItem from "./TabnineTreeItem";
+
+export default class TabnineTreeProvider
+  implements TreeDataProvider<TabnineTreeItem> {
+  onDidChangeTreeData?:
+    | Event<void | TabnineTreeItem | null | undefined>
+    | undefined;
+
+  getTreeItem(element: TabnineTreeItem): TreeItem {
+    return element as TreeItem;
+  }
+
+  getChildren(): ProviderResult<TabnineTreeItem[]> {
```

Comment:
I understand this hook is called when user opens the view - maybe you can use it to open the hub on `home`?
Instead of letting the user click it